api/pom.xml

```diff
@@ -18,8 +18,7 @@
 	</parent>
 
 	<artifactId>kenyaemr-api</artifactId>
-    <version>17.1.0</version>
```

Comment:
Why are you doing away with the version?

Reply:
Just to avoid repetition. The idea is to define the version number once in the main **pom** file and have it inherited by the child **pom** files